package.json

```diff
@@ -204,7 +204,7 @@
     "balena-image-manager": "^7.0.3",
     "balena-preload": "^10.3.1",
     "balena-release": "^3.0.0",
-    "balena-sdk": "^15.3.1",
+    "balena-sdk": "^15.3.7",
```

Comment:
Can you remove pinejs-client-core from the explicit deps as it's no longer directly depended on

Reply:
on it